e2e/suite1/specs/variables/new-query-variable.ts

```diff
@@ -46,10 +46,9 @@ describe('Variables - Add variable', () => {
     e2e.pages.Dashboard.Settings.Variables.Edit.QueryVariable.queryOptionsDataSourceSelect()
       .should('be.visible')
       .within(select => {
-        e2e.components.Select.singleValue().should('have.text', '');
+        e2e.components.Select.singleValue().should('have.text', 'gdev-testdata');
```

Comment:
I guess the default data source is automatically chosen now which makes sense.

Reply:
yes, exactly